packages/storybook/src/stories/ino-nav-menu/ino-nav-menu.scss

```diff
@@ -0,0 +1,13 @@
+.story-ino-nav-menu {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 50px;
+
+  aside {
+    ino-nav-menu {
+      display: flex;
+      width: 150px;
+    }
+  }
```

Comment:
Sadly the active section change can't be seen because the menu gets out of view. Could you try making it sticky within the playground?